services/frontend-v3/src/components/cardVisibilityToggle/CardVisibilityToggleView.jsx

```diff
@@ -4,56 +4,77 @@ import {
   TeamOutlined,
   EyeOutlined,
   WarningOutlined,
+  DownOutlined,
+  UserOutlined,
 } from "@ant-design/icons";
-import { Tooltip, Radio, Popconfirm } from "antd";
+import {
+  Tooltip,
+  Radio,
+  Popconfirm,
+  Menu,
+  Dropdown,
+  Button,
+  message,
+  Select,
+} from "antd";
 import { FormattedMessage } from "react-intl";
 import PropTypes from "prop-types";
+const { Option } = Select;
 
-const CardVisibilityToggleView = ({ status, handleVisibilityToggle, type }) => (
-  <Radio.Group value={status} buttonStyle="solid" size="middle">
-    <Popconfirm
-      title={
-        <FormattedMessage id={`profile.visibility.${type}.show.confirm`} />
-      }
-      placement="topRight"
-      okText={<FormattedMessage id="profile.yes" />}
-      cancelText={<FormattedMessage id="profile.no" />}
-      icon={<WarningOutlined style={{ color: "orange" }} />}
-      onConfirm={() => handleVisibilityToggle("PUBLIC")}
-    >
-      <Tooltip
-        placement="bottom"
-        title={<FormattedMessage id={`profile.visibility.${type}.public`} />}
-      >
-        <Radio.Button value="PUBLIC">
-          <EyeOutlined />
-        </Radio.Button>
-      </Tooltip>
-    </Popconfirm>
-    <Tooltip
-      placement="top"
-      title={<FormattedMessage id={`profile.visibility.${type}.connections`} />}
-    >
-      <Radio.Button
-        value="CONNECTIONS"
-        onClick={() => handleVisibilityToggle("CONNECTIONS")}
-      >
-        <TeamOutlined />
-      </Radio.Button>
-    </Tooltip>
-    <Tooltip
-      placement="top"
-      title={<FormattedMessage id={`profile.visibility.${type}.private`} />}
-    >
-      <Radio.Button
-        value="PRIVATE"
-        onClick={() => handleVisibilityToggle("PRIVATE")}
+const CardVisibilityToggleView = ({ status, handleVisibilityToggle, type }) => {
+  const handleButtonClick = (e) => {
+    message.info("Click on left button.");
+    console.log("click left button", e);
+  };
+
+  const handleMenuClick = (e) => {
+    message.info("Click on menu item.");
+    console.log("click", e);
+  };
+
+  function handleChange(value) {
+    console.log(`selected ${value}`);
+  }
+
+  const menu = (
+    <Menu onClick={handleMenuClick}>
+      <Menu.Item key="PUBLIC" icon={<EyeOutlined />}>
+        Public
+      </Menu.Item>
+      <Menu.Item key="CONNECTIONS" icon={<TeamOutlined />}>
+        Connections
+      </Menu.Item>
+      <Menu.Item key="PRIVATE" icon={<EyeInvisibleOutlined />}>
+        Private
+      </Menu.Item>
+    </Menu>
+  );
+
+  return (
+    <>
+      {/* <Dropdown overlay={menu}>
+        <Button>
+          Button <DownOutlined />
+        </Button>
+      </Dropdown> */}
+      <Select
+        value={status}
+        style={{ width: 120 }}
+        onChange={handleVisibilityToggle}
       >
-        <EyeInvisibleOutlined />
-      </Radio.Button>
-    </Tooltip>
-  </Radio.Group>
-);
+        <Option value="PUBLIC">
+          <EyeOutlined style={{ marginRight: "3px" }} /> Public
```

Comment:
I figured out to set the default "padding" for the icons, you can surround the text following or preceding the icon with span tags. Note, it probably doesn't really affect the padding here since you are having a hard coded string, but when you are using the FormattedMessage from react-intl, it will make a difference.

```suggestion
          <EyeOutlined /> 
          <span>
                Public
           </span>
```

Reply:
thanks! yea the PR is supper beta. i just put it together to get feedback

Reply:
the span isnt working to add padding

Reply:
Are you trying it with hardcoded text or FormattedMessage? I know it works with FormattedMessage (if you search for <span> in the code, you'll see examples)